app/javascript/dashboard/components/ChatList.vue

```diff
@@ -48,19 +29,64 @@
             class="delete-custom-view__button"
             @click="onClickOpenDeleteFoldersModal"
           />
+          <woot-button
+            v-if="!hasActiveFolders"
+            v-tooltip.right="$t('FILTER.TOOLTIP_LABEL')"
+            variant="clear"
+            color-scheme="secondary"
+            icon="filter"
+            size="small"
+            class="btn-filter"
+            @click="onToggleAdvanceFiltersModal"
+          />
         </div>
-
-        <woot-button
-          v-else
-          v-tooltip.right="$t('FILTER.TOOLTIP_LABEL')"
-          variant="clear"
-          color-scheme="secondary"
-          icon="filter"
-          size="small"
-          class="btn-filter"
-          @click="onToggleAdvanceFiltersModal"
+        <switch-layout
+          :is-on-expanded-layout="isOnExpandedLayout"
+          @toggle="$emit('toggle-conversation-layout')"
         />
       </div>
+      <div v-if="!hasActiveFolders" class="filter--actions">
+        <div
+          v-if="!hasAppliedFilters && !hasActiveFolders"
+          class="filter--wrap"
+        >
+          <assignee-type-filter
+            v-if="!hasAppliedFiltersOrActiveFolders"
+            :items="assigneeTabItems"
+            class="chat-status--filter"
+            @filterChange="updateAssigneeTab"
+          />
+          <chat-filter
+            v-if="!hasAppliedFiltersOrActiveFolders"
+            class="chat-status--filter"
+            @statusFilterChange="updateStatusType"
+          />
+          <sort-by-filter
+            v-if="!hasAppliedFiltersOrActiveFolders"
+            @changeSortByFilter="updateSortByFilterType"
+          />
+        </div>
```

Comment:
The condition in the parent and the condition in the child elements evaluate to the same thing
`!A && !B === !(A || B)`

```suggestion
        <div
          v-if="!hasAppliedFiltersOrActiveFolders"
          class="filter--wrap"
        >
          <assignee-type-filter
            :items="assigneeTabItems"
            class="chat-status--filter"
            @filterChange="updateAssigneeTab"
          />
          <chat-filter
            class="chat-status--filter"
            @statusFilterChange="updateStatusType"
          />
          <sort-by-filter
            @changeSortByFilter="updateSortByFilterType"
          />
        </div>
```

Reply:
Done https://github.com/chatwoot/chatwoot/pull/6186/commits/ff39c184c70fbd6ea4db82e5f6767dbd9b2adb74